### PR TITLE
Promote v2 to prod

### PR DESCRIPTION
**Pull request by JasoonS**

### CODE REVIEW

File: docs/HyperIndex/Advanced/async-mode.mdx
Comment (JasoonS) [inline docs/HyperIndex/Advanced/async-mode.mdx:1]: Unused file

File: docs/HyperSync/HyperRPC/hyperrpc-url-endpoints.md [hunk 1]
```diff
@@ -77,3 +77,7 @@ Here is a table of the currently supported networks on HyperRPC and their respec
 
 
 
+
+
+
+
```
Comment (JasoonS) [inline docs/HyperSync/HyperRPC/hyperrpc-url-endpoints.md:83]: Arg, this junk is  fixed here: https://github.com/enviodev/docs/pull/404

Comment (JasoonS) [reply docs/HyperSync/HyperRPC/hyperrpc-url-endpoints.md:83]: Basically a bug in my script every time I run the script it adds a line. Was a mistake to commit it.

File: docusaurus.config.js [hunk 2]
```diff
@@ -93,7 +93,12 @@ const redirectsList = [
   },
   {
     from: "/docs/linked-entity-loaders",
-    to: "/docs/HyperIndex/linked-entity-loaders",
+    // this has been updated to point to v1 since it doesn't exist in v2.
+    to: "/docs/HyperIndex/v1/linked-entity-loaders",
+  },
+  {
+    from: "/docs/HyperIndex/linked-entity-loaders",
+    to: "/docs/HyperIndex/v1/linked-entity-loaders",
```
Comment (JasoonS) [inline docusaurus.config.js:101]: Makes sure old links keep working.

File: docusaurus.config.js [hunk 3]
```diff
@@ -379,12 +392,11 @@ const config = {
         versions: {
           current: {
             label: "v2",
-            path: "v2",
-            banner: "unreleased",
```
Comment (JasoonS) [inline docusaurus.config.js:383]: Remove the banner.

File: docs/HyperIndex/migration-guide-v1-v2.md [hunk 4]
```diff
@@ -5,14 +5,12 @@ sidebar_label: Migration Guide v1 to v2
 slug: /migration-guide-v1-v2
 ---
 
-NOTE: v2 is currently in rc phase (release candidate) and is not yet stable. Please refer to the [v1 documentation](https://docs.envio.dev/docs/HyperIndex) for the stable version.
```
Comment (JasoonS) [inline docs/HyperIndex/migration-guide-v1-v2.md:8]: Yay!

**Review by moose-code — APPROVED** [review]
LGTM

**Pull request merged**